sdk/gossip/client/client.go

```diff
@@ -363,3 +364,7 @@ func (c *Client) SubscribeToAbr(ctx context.Context, abr *services.AddBlockReque
 func (c *Client) UnsubscribeFromAbr(s subscription) {
 	c.subscriber.unsubscribe(s)
 }
+
+func (c *Client) DagGetter() *types.ClientDagGetter {
```

Comment:
these seems like it complects the two types here and that the wrapping should happen at the place that needs to do the wrapping.

Reply:
removed